app/src/shared/errors/Web3/index.jsx

```diff
@@ -54,9 +80,60 @@ export class WrongNetworkSelectedError extends Error {
     }
 }
 
+export class ChallengeFailedError extends Error {
+    code: string
+
+    constructor(message: string = 'challenge failed', ...args: any[]) {
```

Comment:
How about we start messages with capital letters? "Challenge failed", "Create identity failed", etc. Again, cosmetic, skippable suggestion. Cheers!

Reply:
Yea, though I should probably add translations for these.